src/client/trackers/realUserMonitoringForPerformance.js

```diff
@@ -17,7 +17,7 @@ const requiredMetrics = [
 	'totalBlockingTime'
 ];
 
-const samplePercentage = 5;
+const samplePercentage = 10;
```

Comment:
**question**: will increasing this percentage potentially have adverse effects on these users? i seem to remember from our testing that Perfume wasn't the most efficient

Reply:
That's an interesting point - can you tel me more?  What sort of testing?  Do our performance gathering metrics.... affect the page's performance?